src/moments.jl

```diff
@@ -137,6 +137,8 @@ weights used:
 """
 Base.std(v::RealArray, w::AbstractWeights; mean=nothing, corrected::DepBool=nothing) =
     sqrt.(var(v, w; mean=mean, corrected=depcheck(:std, corrected)))
+Base.std(v::AbstractArray{<:AbstractFloat}, w::AbstractWeights; mean=nothing, corrected::DepBool=nothing) =
```

Comment:
Why is this needed? To fix an ambiguity?

Reply:
Yes, see my comments [here](https://github.com/JuliaLang/julia/commit/2cc82d29e1063aad0acc680bb546a5bd475743ee)